### ISSUE TITLE
Multiprocessing + pickling bug.

### ISSUE THREAD

**Issue by drvinceknight** [Axelrod-Python/Axelrod#516]
`python3.5 run_axelrod --noise 0.05 -r 1 -o assets -p 0` gives:

```
Axelrod-tournament(gh-pages) ✗: python3.5 run_axelrod --noise 0.05 -r 1 -o assets -p 0
Starting basic_strategies tournament: Turns: 200, Repetitions: 1, Strategies: 8.
Finished all basic_strategies tasks in 12.2s
Starting ordinary_strategies tournament: Turns: 200, Repetitions: 1, Strategies: 113.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/multiprocessing/queues.py", line 241, in _feed
    obj = ForkingPickler.dumps(obj)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/multiprocessing/reduction.py", line 50, in dumps
    cls(buf, protocol).dump(obj)
_pickle.PicklingError: Can't pickle <class 'axelrod.strategy_transformers.BackStabber'>: attribute lookup BackStabber on axelrod.strategy_transformers failed
```

This issue is a mirror of the issue raised here: https://github.com/Axelrod-Python/tournament/issues/31

**Comment by drvinceknight** [comment]
This has something to do with multiprocessing. The following runs fine:

```
python run_axelrod --noise 0.05 -r 1 -o assets
```

Ie running without multiprocessing.

**Comment by drvinceknight** [comment]
SOs about problems with the multiprocessing module:

- http://stackoverflow.com/questions/8804830/python-multiprocessing-pickling-error
- http://stackoverflow.com/questions/22418816/python-pickling-and-multiprocessing

Looks like most flexible way forward would be to use an alternative multiprocessing solution...

Issue with it with pickling `Backstabber`: https://gist.github.com/marcharper/3b9335ca628fac992ca4a6dfe7cb07e9

**Comment by marcharper** [comment]
The change that causes the issue seems to be the placing of matches in a Queue (causing a pickle event). Commenting line 253 of `tournament.py`:
```
            done_queue.put(new_matches)
```
and running the modified command (to skip the basic tournament):
```
python run_axelrod --noise 0.05 -r 1 -o assets -p 1 --xb
```
avoids the error (but leads to another error since there is no tournament output).

**Comment by drvinceknight** [comment]
Fixed by #517 and #518 has been opened to improve the fix done there (it points back here for the links).
